tests/test_actions.py

```diff
@@ -35,7 +35,6 @@
         (None, None, None, False, "study_valid__etl_table", does_not_raise()),
         (None, None, None, False, "study_valid__nlp_table", does_not_raise()),
         (None, None, None, False, "study_valid__lib_table", does_not_raise()),
-        (None, None, None, False, "study_valid__lib", does_not_raise()),
```

Comment:
Why this line?

Reply:
accidental when trying to insert a new case in this block vs making a new test, restoring.